src/components/CustomKeyboard.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { KeyboardAvoidingView, Platform } from 'react-native';
+import styled from 'styled-components/native';
+
+const Wrapper = styled(KeyboardAvoidingView).attrs({
+  behavior: Platform.OS === 'ios' ? 'padding' : 'height',
```

Comment:
great

Reply:
This is not the best practice i think the keyboard avoinding view is at styled you dont need to import it